doc/manual/installing.rst

```diff
@@ -133,15 +135,15 @@ Installing the host-side software
         $ cd ~/artiq-dev/llvm-or1k
         $ mkdir build
         $ cd ~/artiq-dev/llvm-or1k/build
-        $ ../configure --prefix=/usr/local/llvm-or1k
-        $ make ENABLE_OPTIMIZED=1 -j4
-        $ sudo -E make install ENABLE_OPTIMIZED=1
+        $ cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local/llvm-or1k -DLLVM_TARGETS_TO_BUILD=OR1K -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=ON
```

Comment:
* autoconf is deprecated in LLVM. 
 * why build all targets (the default)? this just wastes time.
 * debug builds finish much quicker for an insignificant (for our purposes) performance loss. also, debug symbols help when you get an assert.
 * shared libs make the install (and any installed dependency as well) take almost an order of magnitude less space. llvm uses rpath so no LD_LIBRARY_PATH alterations are necessary. you also don't have to rebuild llvmlite if you update llvm.